File: docs/components_page/components/dropdown/custom_style.jl
using DashBootstrapComponents

dropdown = dbc_dropdownmenu(
    [
        dbc_dropdownmenuitem("Item 1"),
        dbc_dropdownmenuitem("Item 2"),
        dbc_dropdownmenuitem("Item 3"),
    ],
    label = "Custom",
    className = "m-1",
    toggle_style = Dict("textTransform" => "uppercase", "background" => "#FB79B3"),
    toggleClassName = "fst-italic border border-dark",
)

File: docs/components_page/components/dropdown/custom_style.R
library(dashBootstrapComponents)

dropdown <- dbcDropdownMenu(
  list(
    dbcDropdownMenuItem("Item 1"),
    dbcDropdownMenuItem("Item 2"),
    dbcDropdownMenuItem("Item 3")
  ),
  label = "Custom",
  className = "m-1",
  toggle_style = list(
    "textTransform" = "uppercase",
    "background" = "#FB79B3"
  ),
  toggleClassName = "fst-italic border border-dark"
)
